core/src/main/scala/kafka/server/metadata/AclPublisher.scala

```diff
@@ -1,102 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package kafka.server.metadata
-
-import kafka.utils.Logging
-import org.apache.kafka.image.loader.{LoaderManifest, LoaderManifestType}
-import org.apache.kafka.image.{MetadataDelta, MetadataImage}
-import org.apache.kafka.metadata.authorizer.ClusterMetadataAuthorizer
-import org.apache.kafka.server.authorizer.Authorizer
-import org.apache.kafka.server.fault.FaultHandler
-
-import scala.concurrent.TimeoutException
-
-
-class AclPublisher(
-  nodeId: Int,
-  faultHandler: FaultHandler,
-  nodeType: String,
-  authorizer: Option[Authorizer],
-) extends Logging with org.apache.kafka.image.publisher.MetadataPublisher {
-  logIdent = s"[${name()}] "
```

Comment:
Is not moving this across a small miss or it is unnecessary?

Reply:
Thanks for the suggestion. I change this to log prefix in `LogContext`.